hypothesis-python/src/hypothesis/extra/fakefactory.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, source, providers, locales):
 
     def do_draw(self, data):
         seed = data.draw_bytes(4)
-        random = Random(bytes(seed))
+        random = Random(bytes(seed))  # noqa: B1
```

Comment:
We could use `compat.integer_from_bytes` instead of `bytes` here.

Reply:
In that case I can go a step further and do `data.draw_bits(32)`, and pass the seed directly as a number.